helm_tests/airflow_aux/test_annotations.py

```diff
@@ -426,3 +426,43 @@ def test_precedence(self, values, show_only, expected_annotations):
         for k, v in expected_annotations.items():
             assert k in annotations
             assert v == annotations[k]
+
+
+class TestRedisAnnotations:
+    """Tests Redis Annotations."""
+
+    @pytest.mark.parametrize(
```

Comment:
No need to parametrize if you have a single case. Can you refactor to have these inline in the test?